dbt/models/qc/schema.yml

```diff
@@ -189,35 +189,56 @@ models:
                 - who
                 - wen
               config: *time-filter
+              meta:
+                table_name: asmt
+                description: val01 should not be negative
```

Comment:
**suggestion:** Similar to the fields for `dweldat` it might be handy here to add the corresponding text from the `Asmt` tab of iasWorld. For example, I think this would be:

```suggestion
                description: val01 (FARM LAND) should not be negative
```

Reply:
This is included in the scope of #354.